### ISSUE TITLE
Cannot Open Crates

### ISSUE THREAD

**Issue by TheOtakuJames** [ConnorLinfoot/CratesPlus#592]
Hi there,
When using any crate on my server, with the correct key, all users are told they "Must be holding a [name-of-crate] key!" - even when they are. Attached are 3 images of what I mean - I'm holding a Voters key, and right clicking a Voters crate, and getting told to hold a Voters key. The same happens for my Party crate, and Premium crate - all 3 crates on my server.

MC Version 1.13.2, CratesPlus version 4.5.1

Thanks

![capture](https://user-images.githubusercontent.com/13387337/49191458-88dc1900-f3c6-11e8-8618-c55b23b1ad6b.PNG)
![2018-11-29_11 03 31](https://user-images.githubusercontent.com/13387337/49191467-92658100-f3c6-11e8-8a12-8177c05ed1e0.png)
![2018-11-29_11 03 38](https://user-images.githubusercontent.com/13387337/49191468-92658100-f3c6-11e8-9c4e-dc435afda942.png)

**Comment by MithrandirCraft** [comment]
Yep. Same problem here. Crates Plus just doesn't function propperly on 1.13.2.

**Comment by TheOtakuJames** [comment]
Bump for @ConnorLinfoot

**Comment by MithrandirCraft** [comment]
@TheOtakuJames Hey, make sure you're using the right item id's. I have this problem and I just figured out it can actually work if I stop using the CSGO opener, which is currently broken, and update item names to 1.13.2
